src/vellum/workflows/nodes/displayable/subworkflow_deployment_node/node.py

```diff
@@ -155,14 +155,36 @@ def _compile_subworkflow_inputs_for_direct_invocation(self, workflow: "BaseWorkf
             filtered_inputs = {k: v for k, v in self.subworkflow_inputs.items() if v is not None}
             return inputs_class(**filtered_inputs)
 
+    def _get_release_id(self, deployment_name: str, release_tag: str) -> Optional[str]:
+        """Fetch the release ID for a given deployment name and release tag."""
+        try:
+            release = self._context.vellum_client.workflow_deployments.retrieve_workflow_deployment_release(
+                id=deployment_name, release_id_or_release_tag=release_tag
+            )
```

Comment:
instead of fetching a release here, our existing api fetches should assume the existence of one.

Reply:
In fact, instead of the release id, let's put the hashed module in the raw data